packages/backend-api/src/Resources/config/services.yml

```diff
@@ -1,8 +1,20 @@
 services:
-    Shopsys\BackendApiBundle\Controller\:
-        resource: '../../Controller/'
-        public: false
+    _defaults:
         autowire: true
+        autoconfigure: true
+        public: false
+
+    Shopsys\BackendApiBundle\:
+        resource: '../../**/*{DataSetter,Factory,Transformer,ValidationRunner,Validator}.php'
```

Comment:
It's mandatory to have ValidationRunner as it's not expected to be more of them and Validator as it still needs to be configured manually (because of the interface)